tests/integration/security/sds_vault_flow/sds_vault_flow_test.go

```diff
@@ -52,34 +52,6 @@ const (
 		"U44vy\\\\nw4Nm32yDWhD6AC68rDkXX68m\\\\n-----END CERTIFICATE-----"
 )
```

Comment:
Use ` instead for better raw data format.

Reply:
I tried the backtick multiline string but it does not work since 1)linter does not allow long line. 2) if the long line is splitted into multiple lines using backtick multiline string, it adds a newline char for each line to the original line, which makes the certificate unrecognizable to the recipient. I tried multiple ways to format the certificate and the only one works is the one in the PR.
The example program at https://play.golang.org/p/nnSaeukrSRy demonstrates that if a long line is splitted into multiple lines using backtick multiline string, it adds a newline char for each line to the original line.